test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +42,10 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    @Test
+    public void isAnyNull() {
+        assertTrue(Utils.isAnyNull(null, "abc", "abc"));
+        assertFalse(Utils.isAnyNull("abc", "abd"));
```

Comment:
May be add more test cases for:
- A single null object
- Empty list 
- Multiple null objects etc.